Sources/BrowserServicesKit/Autofill/AutofillUserScript+SecureVault.swift

```diff
@@ -484,19 +480,6 @@ extension AutofillUserScript {
 
         let domain = hostForMessage(message)
 
-        if request.mainType == .credentials, request.subType == .password, let generatedPassword = request.generatedPassword?.value, !generatedPassword.isEmpty {
```

Comment:
@afterxleep this code is needed for password generation on iOS (to notify the native layer to display the autofill prompt and return back the users selected action)

Reply:
you're right.. added this method back @amddg44